cpp/src/IceSSL/SecureTransportTransceiverI.cpp

```diff
@@ -155,37 +155,65 @@ checkTrustResult(SecTrustRef trust,
     UniqueRef<CFErrorRef> trustErr;
     if(trust)
     {
-        if((err = SecTrustSetAnchorCertificates(trust, engine->getCertificateAuthorities())))
-        {
-            throw SecurityException(__FILE__, __LINE__, "IceSSL: handshake failure:\n" + sslErrorToString(err));
-        }
-
-        //
-        // Disable network fetch, we don't want this to block.
-        //
+        // Do not allow to fetch missing intermediate certificates from the network.
```

Comment:
Is this consistent with other SSL implementations? All of them forbids fetching intermediate certificates?

Reply:
OpenSSL doesn't fetch anything, for SChannel seems that the equivalent is `CERT_CHAIN_CACHE_ONLY_URL_RETRIEVAL` but we currently don't set this flag. Note that I change the comment because this doesn't affect OCSP requests